src/handlers/shared/get-user-role.ts

```diff
@@ -2,19 +2,26 @@ import { Context } from "../../types";
 
 export async function getUserRole(context: Context, user: string): Promise<string> {
   const orgLogin = context.payload.organization?.login;
+  const { config } = context;
+  const { maxConcurrentTasks } = config.miscellaneous;
 
   if (!orgLogin) {
     throw new Error("Organization login not found in context payload.");
   }
 
-  try {
```

Comment:
Most if not all other octokit calls are wrapped with a try catch, let's keep this one wrapped too

Reply:
Done Thanks!